src/test/java/kitchenpos/application/MenuGroupServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package kitchenpos.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+import java.util.Arrays;
+import java.util.List;
+import kitchenpos.application.fixture.MenuGroupFixture;
+import kitchenpos.dao.MenuGroupDao;
+import kitchenpos.domain.MenuGroup;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@DisplayName("메뉴 그룹 테스트")
+@ExtendWith(MockitoExtension.class)
+class MenuGroupServiceTest {
+
+    @Mock
+    private MenuGroupDao menuGroupDao;
+
+    @InjectMocks
+    private MenuGroupService menuGroupService;
+
+    private MenuGroup 메뉴그룹1;
+    private MenuGroup 메뉴그룹2;
+
+    @BeforeEach
+    void setUp() {
+        메뉴그룹1 = MenuGroupFixture.create(1L, "두마리메뉴");
```

Comment:
```suggestion
        두마리메뉴 = MenuGroupFixture.create(1L, "두마리메뉴");
```

---

클린코드 25p

의미 있게 구분하라
컴파일러나 인터프리터만 통과하려는 생각으로 코드를 구현하는 프로그래머는 스스로 문제를 일으킨다.
예를 들어, 동일한 범위 안에서 다른 두 개념에 같은 이름을 사용하지 못한다.
...

컴파일러를 통과할지라도 연속된 숫자를 덧붙이거나 불용어(noise word)를 추가하는 방식은 적절하지 못하다.
이름이 달라야 한다면 의미도 달라져야 한다.

연속적인 숫자를 덧붙인 이름(a1, a2 ... aN)은 의도적인 이름과 정반대다.
이런 이름은 그릇된 정보를 제공하는 이름도 아니며, 아무런 정보를 제공하지 못하는 이름일 뿐이다.
저자 의도가 전혀 드러나지 않는다.